### ISSUE TITLE
User Guide Plot

### ISSUE THREAD

**Issue by ScrapeWithYuri** [ZhengyaoJiang/PGPortfolio#124]
Is the user guide plot still accurate? My tests are generally showing break-even return.

**Comment by ScrapeWithYuri** [comment]
The config I am using is this

`{
  "agent_type": "NNAgent",
  "input": {
    "coin_number": 11,
    "coins": [
      "ETH",
      "XRP",
      "BCH",
      "LTC",
      "STR",
      "XMR",
      "DASH",
      "ETC",
      "XEM",
      "NXT",
      "EMC2"
    ],
    "end_date": "2019/06/01",
    "fake_ratio": 1,
    "feature_number": 3,
    "global_period": 1800,
    "is_permed": false,
    "market": "poloniex",
    "norm_method": "absolute",
    "online": false,
    "portion_reversed": false,
    "save_memory_mode": false,
    "start_date": "2016/01/01",
    "test_portion": 0.08,
    "volume_average_days": 30,
    "window_size": 30
  },
  "layers": [
    {
      "activation_function": "relu",
      "filter_number": 2,
      "filter_shape": [
        1,
        3
      ],
      "padding": "valid",
      "regularizer": null,
      "strides": [
        1,
        1
      ],
      "type": "ConvLayer",
      "weight_decay": 0
    },
    {
      "activation_function": "relu",
      "filter_number": 10,
      "regularizer": "L2",
      "type": "EIIE_Dense",
      "weight_decay": 5e-9
    },
    {
      "regularizer": "L2",
      "type": "EIIE_Output_WithW",
      "weight_decay": 5e-8
    }
  ],
  "random_seed": 0,
  "trading": {
    "buffer_biased": 0.00005,
    "learning_rate": 0.00028,
    "rolling_training_steps": 85,
    "trading_consumption": 0.0025
  },
  "training": {
    "batch_size": 160,
    "buffer_biased": 0.00005,
    "decay_rate": 1,
    "decay_steps": 50000,
    "fast_train": true,
    "learning_rate": 0.00028,
    "loss_function": "loss_function6",
    "snap_shot": false,
    "steps": 80000,
    "training_method": "GradientDescent"
  }
}`

And here are my results.

![Figure_1](https://user-images.githubusercontent.com/47489191/67051457-f7fe5100-f0ef-11e9-8959-00395fca50d0.png)

**Comment by dexhunter** [comment]
it seems to me that the agent is stuck on a local minimum or fail to initialize
